src/Collection/CollectionTrait.php

```diff
@@ -819,7 +819,7 @@ protected function optimizeUnwrap()
     {
         $iterator = $this->unwrap();
 
-        if ($iterator instanceof ArrayIterator) {
+        if (get_class($iterator) === ArrayIterator::class && $iterator instanceof ArrayIterator) {
```

Comment:
Isn't this redundant?

Reply:
Very redundant, but otherwise PHPStan fails with a `Call to an undefined method Traversable::getArrayCopy()` error.

Reply:
We can just ignore the error in phpstan.neon instead of adding such redundant check.

Reply:
This looks like probably an error in phpstan and should be reported.